<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindResDlg</class>
 <widget class="QDialog" name="FindResDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>186</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <hintingpreference>PreferDefaultHinting</hintingpreference>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stack_wid">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="success_page">
      <property name="minimumSize">
       <size>
        <width>280</width>
        <height>186</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>280</width>
        <height>186</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="sizeConstraint">
        <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
       </property>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QGridLayout" name="gridLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="headshot_lab">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>60</height>
             </size>
            </property>
            <property name="text">
             <string>Pixmap</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="name_lab">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <hintingpreference>PreferDefaultHinting</hintingpreference>
             </font>
            </property>
            <property name="text">
             <string>name</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="desc_lab">
            <property name="text">
             <string>dsc</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="sex_lab">
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="resource.qrc">:/res/male.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>dsc</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="sizeConstraint">
           <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
          </property>
          <item>
           <widget class="ClickBtn" name="addto_contracts_btn">
            <property name="minimumSize">
             <size>
              <width>110</width>
              <height>35</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>110</width>
              <height>35</height>
             </size>
            </property>
            <property name="text">
             <string>添加到通讯录</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="failed_page">
      <property name="minimumSize">
       <size>
        <width>260</width>
        <height>160</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>260</width>
        <height>160</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
       </property>
       <item alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>100</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <hintingpreference>PreferDefaultHinting</hintingpreference>
          </font>
         </property>
         <property name="text">
          <string>无法找到该用户，请检查你填写的账号是否正确。</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="ClickBtn" name="notfound_btn">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <hintingpreference>PreferDefaultHinting</hintingpreference>
          </font>
         </property>
         <property name="text">
          <string>确认</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickBtn</class>
   <extends>QPushButton</extends>
   <header location="global">clickbtn.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
